### PR TITLE
fix(api): few tests were flakey in the ci/cd

### PR DESCRIPTION
**Pull request by p-fernandez**
### What change does this PR introduce?

There was some flakiness in a pair of tests that failed in the CI/CD and locally. Individually the tests were passing.

### Why was this change needed?

Clean the error reporting in the CI/CD.

### CODE REVIEW

File: apps/api/src/app/events/e2e/send-message-push.e2e.ts [hunk 1]
```diff
@@ -39,7 +41,7 @@ describe('Trigger event - Send Push Notification - /v1/events/trigger (POST)', (
   });
 
   after(() => {
-    process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = 'false';
+    process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = ORIGINAL_IS_MULTI_PROVIDER_CONFIGURATION_ENABLED;
```
Comment (p-fernandez) [inline apps/api/src/app/events/e2e/send-message-push.e2e.ts:44]: To keep consistency on the default state of the system for this feature flag.

File: apps/api/src/app/events/e2e/trigger-event.e2e.ts [hunk 2]
```diff
@@ -2390,7 +2391,8 @@ describe(`Trigger event - ${eventTriggerPath} (POST)`, function () {
       });
     });
   });
-  describe.skip('Trigger Event - [IS_MULTI_PROVIDER_CONFIGURATION_ENABLED=true]', function () {
+
+  describe('Trigger Event - [IS_MULTI_PROVIDER_CONFIGURATION_ENABLED=true]', function () {
```
Comment (p-fernandez) [inline apps/api/src/app/events/e2e/trigger-event.e2e.ts:2395]: Not needed to skip anymore.

File: apps/api/src/app/events/e2e/send-message-push.e2e.ts (hunk 1 above)
Comment (djabarovgeorge) [reply apps/api/src/app/events/e2e/send-message-push.e2e.ts:44]: yeah, that should help us do more consistent e2e on every test.
did you have the chance to see that after this change the flakiness faded?

Comment (p-fernandez) [reply apps/api/src/app/events/e2e/send-message-push.e2e.ts:44]: Yes, both locally and in the CI. Actually the checks passed at first try!

**Pull request merged**